src/main/java/DukeException.java

```diff
@@ -0,0 +1,2 @@
+public class DukeException extends Exception {
+}
```

Comment:
Exploit the fact that you are creating a custom exceptions? e.g., you can consider adding a custom message to your custom exception.